catalog-info.yaml

```diff
@@ -0,0 +1,57 @@
+# Declare a Backstage Component that represents your application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: go-structform
+
+spec:
+  type: library
+  owner: group:ingest-fp
+  system: platform-ingest
+  lifecycle: production
+
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-go-structform
+  description: Buildkite pipeline for the go-structform project
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/go-structform
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: go-structform
+      description: Buildkite pipeline for the go-structform project
+    spec:
+      repository: elastic/go-structform
+      pipeline_file: ".buildkite/pipeline.yml"
+      maximum_timeout_in_minutes: 60
+      provider_settings:
+        build_pull_request_forks: false
+        build_pull_requests: true # requires filter_enabled and filter_condition settings as below when used with buildkite-pr-bot
+        build_tags: true
+        filter_enabled: true
+        filter_condition: >-
+          build.pull_request.id == null || (build.creator.name == 'elasticmachine' && build.pull_request.id != null)
+      cancel_intermediate_builds: true
+      cancel_intermediate_builds_branch_filter: '!main'
+      skip_intermediate_builds: true
+      skip_intermediate_builds_branch_filter: '!main'
+      env:
+        ELASTIC_PR_COMMENTS_ENABLED: 'true'
+      teams:
+        ingest-fp:
+          access_level: MANAGE_BUILD_AND_READ
+        everyone:
+          access_level: READ_ONLY
```

Comment:
let's ensure that we always add the platform ingest eng prod team too like https://github.com/elastic/logstash/blob/8fa3bd0d7f4eed8f808120706a0f74ec5c1847a7/catalog-info.yaml#L265-L266